### ISSUE TITLE
Update documentation on main page

### ISSUE THREAD

**Issue by sergeytrasko** [ctco/cukes#111]
For some time already the project is evolved from 'cukes-rest' to 'cukes'.
It's not about only REST now, it's bigger.

So the documentation on main page (readme.md) should be aligned accordingly.

**Comment by berndgoetz** [comment]
Yes, that makes sense. Plus I'd like to see a documentation or a pointer to a class that contains all Cukes properties descriptions, such as a description of all possible values for `cukes.logging.http.responses.include`.
